src/angular-google-analytics.js

```diff
@@ -187,13 +192,29 @@
         };
 
         var _ga = function() {
+          var tmpCommand, tmpArguments;
```

Comment:
You don't need these defined in this scope as they are only used within the anonymous function passed into the forEach on line 208.